docs/maintaining/tasks.md

```diff
@@ -2,20 +2,20 @@
 
 ## Merging Pull Requests
 
-To build new images on Docker Cloud and publish them to the Docker Hub registry, do the following:
+To build new images and publish them to the Docker Hub registry, do the following:
 
 1. Make sure GitHub Actions status checks pas for the PR.
 2. Merge the PR.
-3. Monitor the Docker Cloud build status for each of the stacks, starting with
-   [jupyter/base-notebook](https://cloud.docker.com/app/jupyter/repository/docker/jupyter/base-notebook/general)
+3. Monitor the Docker Hub build status for each of the stacks, starting with
+   [jupyter/base-notebook](https://hub.docker.com/app/jupyter/repository/docker/jupyter/base-notebook/general)
    and ending with
-   [jupyter/all-spark-notebook](https://cloud.docker.com/app/jupyter/repository/docker/jupyter/all-spark-notebook/general).
+   [jupyter/all-spark-notebook](https://hub.docker.com/app/jupyter/repository/docker/jupyter/all-spark-notebook/general).
```

Comment:
@parente please, check the links here, I don't have any access, so there might be a chance that these links are working for you, but not for me.

Reply:
The links should be like this I think: https://hub.docker.com/r/jupyter/base-notebook/builds

Reply:
I updated this section.
Also changed the name of the PR.